media/js/uelc_admin/uelc_facilitator.js

```diff
@@ -13,6 +14,13 @@ UELCAdmin = {
             this.deleteLibraryItem();
             */
         };
+        this.setFormClickHandler = function() {
+            var btn = jQuery('.gate-block.active .gate-button form .btn');
+            btn.css('cursor', 'pointer');
+            btn.click(function() {
+                jQuery(this).parent().parent('form').submit();
```

Comment:
For finding a surrounding element, I like using `jQuery.closest` or `jQuery.parents` methods .. they can be more stable in case the DOM structure changes. So this line could be changed to `jQuery(this).closest('form').submit()`.